### ISSUE TITLE
Intermittent Failure? Create an XML Import import records through a form upload

### ISSUE THREAD

**Issue by mark-dce** [curationexperts/epigaea#259]
commit: 1bc54457a15c7f89e57893c47fb86483034ecc6d
seed: 62498

```
  1) Create an XML Import import records through a form upload
     Failure/Error: delegate :created_at, :id, :items, to: :object
     
     ActionView::Template::Error:
       BatchPresenter#id delegated to object.id, but object is nil: #<BatchPresenter:0x0000001c6ac3b0 @object=nil>
     # ./app/presenters/batch_presenter.rb:37:in `rescue in id'
     # ./app/presenters/batch_presenter.rb:37:in `id'
     # ./app/presenters/xml_import_presenter.rb:32:in `id'
     # ./app/views/hyrax/batches/_batch_info.html.erb:3:in `_app_views_hyrax_batches__batch_info_html_erb__2812106075327537770_238541300'
     # ./app/views/hyrax/xml_imports/show.html.erb:19:in `_app_views_hyrax_xml_imports_show_html_erb___2491439204201198964_212677300'
     # ------------------
     # --- Caused by: ---
     # NoMethodError:
     #   undefined method `id' for nil:NilClass
     #   ./app/presenters/batch_presenter.rb:37:in `id'
```

**Comment by no-reply** [comment]
another seed: 38569

**Comment by no-reply** [comment]
I can repeat this locally with the seeds given.

However, if I drop a debug point in where the failure arises, it evaporates. This suggests the failure depends at least partially on timing.

**Comment by no-reply** [comment]
Closing this as (probably?) fixed by #270.
